src/machines/modelingMachine.ts

```diff
@@ -3326,11 +3328,7 @@ export const modelingMachine = setup({
             'Artifact graph emptied': 'hidePlanes',
           },
 
-          entry: [
-            'show default planes',
-            'reset camera position',
-            'set selection filter to curves only',
-          ],
+          entry: ['show default planes', 'set selection filter to curves only'],
```

Comment:
I don't know why showing planes moves the camera?

I deleted this because after we initialize the scene the initial camera view code fights this because this does `reset camera position`

Reply:
You're right it shouldn't. Idk if this could be related to @andrewvarga's work on the plane toggles?